.github/workflows/post-release.yaml

```diff
@@ -94,12 +94,17 @@ jobs:
           git config --global user.email "noreply@github.com"
           git config --global user.name "GitHub"
 
+          # get Go version from go.mod
+          GO_VERSION=$(go mod edit -json | jq -r .Go)
```

Comment:
nit: this can be done with just the `go` command: `go list -m -f '{{.GoVersion}}'`. Just FYI

Reply:
I can change it if you feel strongly, I just want to see this run on a real v17 release before I backport to other branches.

Reply:
I do not feel strongly. The abstract concern is that `jq` may not always be installed - self-hosted runners, container runners, etc. But in reality not an issue for this workflow.

Reply:
This entire purpose of this workflow is to update fields in config.json, so if we wanted to handle a missing `jq` we'd have a lot more to worry about than just this one line :-)

Reply:
Yeah, sorry. Tunnel vision on my part. I just looked at that one line :)